crates/ruma-client-api/src/discovery/get_authentication_issuer.rs

```diff
@@ -0,0 +1,48 @@
+//! `GET /_matrix/client/*/auth_issuer`
+//!
+//! Get the OpenID Connect Provider that is trusted by the homeserver.
+
+pub mod msc2965 {
+    //! `MSC2965` ([MSC])
+    //!
+    //! [MSC]: https://github.com/matrix-org/matrix-spec-proposals/pull/2965
+
+    use ruma_common::{
+        api::{request, response, Metadata},
+        metadata,
+    };
+
+    const METADATA: Metadata = metadata! {
+        method: GET,
+        rate_limited: false,
+        authentication: None,
+        history: {
+            unstable => "/_matrix/client/unstable/org.matrix.msc2965/auth_issuer",
+        }
+    };
+
+    /// Request type for the `auth_issuer` endpoint.
+    #[request(error = crate::Error)]
+    pub struct Request {}
```

Comment:
I think we derive `Default` for most / all empty request types?

Reply:
Yes on most of them :slightly_smiling_face: